### PR TITLE
Update matrix-sdk dependency to 75683d268fc2e

### PR DESCRIPTION
**Pull request by andybalaam**
Fixes https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/issues/162

Note: the individual commits don't actually compile. I found it helpful to separate them out to get my head around the changes, but if we want a clean commit history that always compiles, we should squash them. I actually prefer to keep them separate in this case, but I am persuadable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # UNRELEASED
 
+-   Update matrix-rust-sdk to `75683d268fc2e`
```
Comment (richvdh) [inline CHANGELOG.md:3]:
```suggestion
-   Update matrix-rust-sdk to `75683d268fc2e`.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+
+**BREAKING CHANGES**
+
+-   Remove `SignedCurve25519` variant of `DeviceKeyAlgorithm`.
+
+-   One-time keys and fallback keys are now represented by `OneTimeKeyAlgorithm`
+    instead of `DeviceKeyAlgorithm`.
```
Comment (richvdh) [inline CHANGELOG.md:10]: `OneTimeKeyAlgorithm` is not exposed by these bindings, is it?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,14 @@
 # UNRELEASED
 
+-   Update matrix-rust-sdk to `75683d268fc2e`
+
+**BREAKING CHANGES**
+
+-   Remove `SignedCurve25519` variant of `DeviceKeyAlgorithm`.
+
+-   One-time keys and fallback keys are now represented by `OneTimeKeyAlgorithm`
```
Comment (richvdh) [inline CHANGELOG.md:9]:
```suggestion
-   The algorithms of one-time keys and fallback keys are now represented by `OneTimeKeyAlgorithm`
```

File: CHANGELOG.md (hunk 2 above)
Comment (andybalaam) [reply CHANGELOG.md:10]: Hmm, no, but it still seems worth talking about. What do you think?

Comment (andybalaam) [reply CHANGELOG.md]: Removed in c159ace

**Review by richvdh — APPROVED** [review]

**Pull request merged**